content/en/docs/install-operate/running/index.md

```diff
@@ -221,6 +275,8 @@ docker run --rm -i -t \
 # if you are running a kernel version < 4.14
 ```
 
+_________________
```

Comment:
?

Reply:
That's because i didn't want to add a whole new section for running `falco-no-driver` in fully privileged mode (the section that starts right after), and at the same time i didn't want to attach this section to the `eBPF` fully privileged mode (the section in which we are in), therefore i added a small divider.